### PR TITLE
adjusted logo size dashboard

### PR DESCRIPTION
**Pull request by SjoerdHilhorst**

### CODE REVIEW

File: src/sources/forus-platform/scss/_common/landing.scss [hunk 1]
```diff
@@ -1212,9 +1212,8 @@ ui-view {
 
             img {
                 display: block;
-                margin: 0 0;
-                height: 50px;
-                margin: 0;
+                height: 20px;
+                margin: 15px 0px auto auto;
```
Comment (GerbenBosschieter) [inline src/sources/forus-platform/scss/_common/landing.scss]: Screen Shot 2020-06-04 at 10 00 08
Remove margin. This results into weird result on mobile.

File: src/sources/forus-platform/scss/_common/landing.scss [hunk 2]
```diff
@@ -1212,9 +1212,8 @@ ui-view {
 
             img {
                 display: block;
-                margin: 0 0;
-                height: 50px;
-                margin: 0;
+                height: 20px;
```
Comment (GerbenBosschieter) [inline src/sources/forus-platform/scss/_common/landing.scss]: To small. If. choose 30 it's okay.

File: src/sources/forus-platform/scss/_common/landing.scss [hunk 3]
```diff
@@ -2167,7 +2166,7 @@ ui-view {
     @media screen and (max-width: 1700px) {
         &.block-navbar {
             .navbar-logo {
-                padding: 20px 20px 20px 0;
+                padding: 20px 25px 20px 0;
```
Comment (GerbenBosschieter) [inline src/sources/forus-platform/scss/_common/landing.scss]: In combination with my earlier feedback, adding this much padding: 30px 30px should do the job.

**Review by GerbenBosschieter — CHANGES_REQUESTED** [review]
@SjoerdHilhorst I'have left some comments.
I can see that you made some changes to the common file.
Changing css in the common file can result into unintentionally changes to other implementations.

I tried to check if other implementations where influenced by the changes you made. Didn't look like it had any issues. But couldn't check everything

**Comment by maxvisser** [comment]
@GerbenBosschieter Will you review this again?

**Pull request merged**